### PR TITLE
Update Dependencies

### PR DESCRIPTION
**Pull request by MatthewCallis**
Update Dependencies, Raven was reporting itself as being out of date inside of Sentry, and went ahead and updated everything else while I was in there.

[raven v1.1.1 ➜ v2.0.2](https://github.com/getsentry/raven-node/blob/master/History.md)
babel-cli  v6.18.0 ➜ v6.24.1
babel-core v6.21.0 ➜ 6.24.1
babel-eslint v7.1.0 ➜ v7.2.3
babel-jest v18.0.0 ➜ v20.0.3
babel-preset-es2015 v6.28.0 ➜ v6.24.1
[eslint v3.13.1 ➜ v3.19.0](https://github.com/eslint/eslint/blob/master/CHANGELOG.md)
eslint-config-airbnb-base v11.0.1 ➜ v11.2.0
[eslint-plugin-import v2.2.0 ➜ v2.3.0](https://github.com/benmosher/eslint-plugin-import/blob/master/CHANGELOG.md)
[jest v18.1.0 ➜ v20.0.4](https://github.com/facebook/jest/blob/master/CHANGELOG.md)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -46,6 +46,6 @@
   "license": "MIT",
   "dependencies": {
     "hoek": "~4.1.0",
-    "raven": "~1.1.1"
+    "raven": "~2.0.2"
```
Comment (jsynowiec) [inline package.json:49]: Can you please split this into two separate commits? One with dev dependencies and the other with production ones.

( Personally I like to keep packages grouped while updating, e.g. bump babel-* versions, test and commit, then other packages. It's easier to verify and revert if there are problems. )

**Review by jsynowiec — CHANGES_REQUESTED** [review]
Thanks for contribution!

**Comment by MatthewCallis** [comment]
@jsynowiec I split the commits up, 👍🏼

File: package.json (hunk 1 above)
Comment (MatthewCallis) [reply package.json:49]: Done!

**Review by jsynowiec — APPROVED** [review]

**Pull request merged**